.github/workflows/build.yaml

```diff
@@ -95,7 +95,7 @@ jobs:
     strategy:
       matrix:
         package: ${{ fromJSON(needs.changes.outputs.packages) }}
-        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
```

Comment:
we can probably drop 3.7 support soon due to end-of-life reaching

Reply:
yeah, I guess